src/dune_rules/merlin/merlin.ml

```diff
@@ -51,17 +51,22 @@ module Processed = struct
     ; src_dirs : Path.Set.t
     ; flags : string list
     ; extensions : string option Ml_kind.Dict.t list
+    ; extension_to_reader : string list String.Map.t
```

Comment:
Shall we call this field `readers` as well?

Reply:
Yes, sorry, just pushed a commit fixing this.